nodeEditor/src/serializationTools.ts

```diff
@@ -22,14 +28,14 @@ export class SerializationTools {
             });
         }
 
-        globalState.storeEditorData(material.editorData);
+        globalState.storeEditorData(material.editorData, frameId);
     }
 
-    public static Serialize(material: NodeMaterial, globalState: GlobalState, selectedBlocks?: NodeMaterialBlock[]) {
+    public static Serialize(material: NodeMaterial, globalState: GlobalState, selectedBlocks?: NodeMaterialBlock[], frameId?: number) {

```

Comment:
maybe we can simplify by adding a nullable frame (instead of giving the frame.id and the frame content)?